src/commcare_cloud/ansible/roles/commcarehq/tasks/celery.yml

```diff
@@ -61,3 +58,38 @@
         no_proxy: "{% if http_proxy_address is defined %}{{ groups['all'] | join(',') }},{{ app_processes_config.additional_no_proxy_hosts }}{% endif %}"
         PROMETHEUS_PUSHGATEWAY_HOST: "{% if prometheus_celery_pushgateway is defined %}{{ prometheus_celery_pushgateway }}{% endif %}"
         TMPDIR: '{{ encrypted_tmp }}'
+
+- name: add "DigiCert SHA2 Secure Server CA" certificate to certifi
+  blockinfile:
+    path: "{{ py3_virtualenv_home }}/lib/python3.6/site-packages/certifi/cacert.pem"
```

Comment:
For my own understanding, this task will add an extra CA to this file, but should not affect any other CAs (of which I'm guessing there are many) in that file?

At first I thought it was replacing the file, which seemed alarming since that would break SSL for many sites. Do I have that right?

Reply:
Should we add a task to a backlog somewhere to revisit and verify
- this is still needed
- the cert is still valid and has not been compromised

Unfortunately I'm not sure we have good procedural support to ensure that type of task actually gets done.

Reply:
> For my own understanding, this task will add an extra CA to this file, but should not affect any other CAs (of which I'm guessing there are many) in that file?

It adds an intermediate certificate signed by a CA root certificate we already trust (it's included by default), without affecting the others.

> At first I thought it was replacing the file, which seemed alarming since that would break SSL for many sites. Do I have that right?

Yes, it just adds that block, it does not replace the file, which would indeed break TLS connections to all sites using certificates signed by a different intermediate or root certificate.

> Should we add a task to a backlog somewhere to revisit and verify
> 
>     * this is still needed
> 
>     * the cert is still valid and has not been compromised
> 
> 
> Unfortunately I'm not sure we have good procedural support to ensure that type of task actually gets done.

I think that's a good idea. Ideally we want to keep this in place only until smspro.mtn.ci fixes their configuration.